common/SmokeTests/SmokeTest/Program.cs

```diff
@@ -191,11 +191,22 @@ void ProcessType(Type type)
                 // Expected; this is a known scenario where the type is not impactful to the
                 // Azure SDK experience and cane be safely ignored.
             }
+            catch (NotSupportedException ex) when (ShouldIgnoreNotSupportedException(ex))
+            {
+                // Expected; this is a known scenario where the type is not able to be created
+                // via reflection using the Activator.  These are not types related to the SDK
+                // and are not impactful to the Azure SDK experience.
+            }
             catch (COMException)
             {
                 // Expected; this is a known scenario where the type is not impactful to the
                 // Azure SDK experience and cane be safely ignored.
             }
+            catch (PlatformNotSupportedException)
+            {
+                // Expected; this is a known scenario where the type is not impactful to the
```

Comment:
Curious - what is the known scenario?

Reply:
The Microsoft test platform library that requires COM and gets angry when run on "not Windows"